### DISCUSSION TITLE
HA Timed out connecting

### DISCUSSION THREAD

**Discussion by JohannCR** [timmo001/system-bridge#1850] [Help] [answered]
### Description

Now that I successfully installed system-bridge with the right python version, and waited for the 2022.7.3 HA update using systembridgeconnector 3.3.2, it still doesn't connect. Fails on the integration config.
Used the system bridge's device (fixed) IP and api-key the command generated. Did not change default port.
Also tried to open 9170 and 9172 ports on local firewall, to no avail.
What else can I do ? Starting to get tiresome...

### What Platform / OS are you running?

Linux

### What version are you running?

3.3.4

**Comment by timmo001** [comment, marked as answer]
Raised here:

https://github.com/home-assistant/core/issues/75532
